DOCS: Rumble support for iOS and Android (#2420)
Unity-Technologies/InputSystem: Packages/com.unity.inputsystem/Documentation~/Gamepad.md

>* PS4, Xbox, and Switch controllers, when connected to their respective consoles. Only supported if you install console-specific input packages in your Project.
>* PS4 controllers, when connected to Mac or Windows/UWP computers.
>* Xbox controllers on Windows.
>* Android:
>    * Note: Rumble automatically stops about 10 seconds after the last SetMotorSpeeds call unless explicitly stopped.
>    * Requires Unity 6000.6 or later
>    * Android 12 or later
>        * Most bluetooth-connected gamepads with rumble support are supported (PS4/PS5 controllers, Xbox controllers, etc.)
>        * __Note:__ If the gamepad is connected via USB cable, rumble support may vary.
>        * __Note:__ If the gamepad supports more than one motor, each motor's strength can be controlled individually.
>    * Android 11 or earlier
>        * Rumble support is limited.
>        * __Note:__ Only one motor is supported. If the gamepad has more than one motor, the higher value of the left and right motor strengths is applied to both.

> >        * __Note:__ In some cases, frequency control is limited, meaning the rumble strength can only be either 1.0 or 0.0.

Reviewer: ```suggestion
>* Gamepads on Android. Rumble support varies with Android OS version and requires Unity 6000.6 or later.
>    * Rumble automatically stops about 10 seconds after the last [`SetMotorSpeeds`](xref:UnityEngine.InputSystem.Haptics.IDualMotorRumble) call unless you explicitly stop it.
>    * Android 12 or later:
>        * Supports most Bluetooth-connected gamepads with rumble support including PS4, PS5, and Xbox controllers.
>        *  If the gamepad is connected via USB cable, rumble support might vary.
>        *  If the gamepad supports more than one motor, you can control the speed of each motor individually.
>    * Android 11 or earlier:
>        * Rumble support is limited.
>        * Only one motor is supported. If the gamepad has more than one motor, the higher value of the left and right motor speed applies to both.
>        * On some devices, motor speed control is limited. You can only set the left and right motor speed to either `1.0`(maximum speed) or `0.0` (turned off).
```

Reviewer: - Have removed nested notes.
- Given there will be some content for iOS too, this note will grow bigger so might make sense to move both Android and iOS content out of the note under their own heading may be "Rumble support on mobile platforms" just after the note (with a mention in the main note as Gamepads on Android and iOS)
- On first bullet point "Rumble automatically stops about 10 seconds after the last `SetMotorSpeeds` call unless you explicitly stop it." which method can users use to stop it explicitly?

Author: @anjalin14 to stop rumble you do SetMotorSpeeds(0, 0)

Reviewer: Thanks, makes sense:). Also, any thoughts on moving the Android/iOS content out of the note into a separate section? might be easier to decide when we've the iOS content.

Author: My personal opinion, that we should do it for all or for none platforms, for consistency sake, but I am not part of input team, so I'll let @jfreire-unity decide :)

>* PS4, Xbox, and Switch controllers, when connected to their respective consoles. Only supported if you install console-specific input packages in your Project.
>* PS4 controllers, when connected to Mac or Windows/UWP computers.
>* Xbox controllers on Windows.
>* Gamepads on Android. Rumble support varies with Android OS version and requires Unity 6000.6 or later.
>    * Rumble automatically stops about 10 seconds after the last [`SetMotorSpeeds`](xref:UnityEngine.InputSystem.Haptics.IDualMotorRumble) call unless you explicitly stop it.
>    * Android 12 or later:
>        * Supports most Bluetooth-connected gamepads with rumble support including PS4, PS5, and Xbox controllers.

> >        *  If the gamepad is connected via USB cable, rumble support might vary.

Reviewer 2: ```suggestion
>        *  If the gamepad is connected with a USB cable, rumble support might vary.
```

>        *  If the gamepad supports more than one motor, you can control the speed of each motor individually.
>    * Android 11 or earlier:
>        * Rumble support is limited.
>        * Only one motor is supported. If the gamepad has more than one motor, the higher value of the left and right motor speed applies to both.

> >        * On some devices, motor speed control is limited. You can only set the left and right motor speed to either `1.0`(maximum speed) or `0.0` (turned off).

Reviewer 2: ```suggestion
>        * On some devices, motor speed control is limited. You can only set the left and right motor speed to either `1.0` (maximum speed) or `0.0` (turned off).
```

> * Gamepads on iOS, tvOS, and visionOS. Requires Unity 6000.6 or later.
>    * Rumble is supported on gamepads recognized by Apple's GameController framework, including PS4, PS5, Xbox, and Switch Pro controllers.
>    * If the gamepad exposes both motors, you can control each speed individually. Otherwise, the higher value of the left and right motor speed applies to all motors.
>    * The first [`SetMotorSpeeds`](xref:UnityEngine.InputSystem.Haptics.IDualMotorRumble) call allocates rumble resources, which adds latency.

> >    * When all motor speeds are set to `0.0`, an inactivity timer starts. After 2 minutes, rumble resources are released to preserve controller battery. The next [`SetMotorSpeeds`](xref:UnityEngine.InputSystem.Haptics.IDualMotorRumble) call reallocates them and again incurs allocation latency.

Reviewer 2: ```suggestion
>    * When all motor speeds are set to `0.0`, an inactivity timer starts. After two minutes, rumble resources are released to preserve controller battery. The next [`SetMotorSpeeds`](xref:UnityEngine.InputSystem.Haptics.IDualMotorRumble) call reallocates them and again incurs allocation latency.
```

> >* PS4, Xbox, and Switch controllers, when connected to their respective consoles. Only supported if you install console-specific input packages in your Project.

Reviewer: ```suggestion
>* PS4, Xbox, and Nintendo Switch controllers, when connected to their respective consoles. Only supported if you install console-specific input packages in your Project.
```

>* PS4 controllers, when connected to Mac or Windows/UWP computers.
>* Xbox controllers on Windows.
>* Gamepads on Android. Rumble support varies with Android OS version and requires Unity 6000.6 or later.
>    * Rumble automatically stops about 10 seconds after the last [`SetMotorSpeeds`](xref:UnityEngine.InputSystem.Haptics.IDualMotorRumble) call unless you explicitly stop it.
>    * Android 12 or later:
>        * Supports most Bluetooth-connected gamepads with rumble support including PS4, PS5, and Xbox controllers.
>        *  If the gamepad is connected with a USB cable, rumble support might vary.
>        *  If the gamepad supports more than one motor, you can control the speed of each motor individually.
>    * Android 11 or earlier:
>        * Rumble support is limited.
>        * Only one motor is supported. If the gamepad has more than one motor, the higher value of the left and right motor speed applies to both.
>        * On some devices, motor speed control is limited. You can only set the left and right motor speed to either `1.0` (maximum speed) or `0.0` (turned off).
> * Gamepads on iOS, tvOS, and visionOS. Requires Unity 6000.6 or later.

> >    * Supports rumble on gamepads that Apple's GameController framework recognizes, including PS4, PS5, Xbox, and Switch Pro controllers.

Reviewer: ```suggestion
>    * Supports rumble on gamepads that Apple's GameController framework recognizes, including PS4, PS5, Xbox, and Nintendo Switch Pro controllers.
```

>        *  If the gamepad is connected with a USB cable, rumble support might vary.
>        *  If the gamepad supports more than one motor, you can control the speed of each motor individually.
>    * Android 11 or earlier:
>        * Rumble support is limited.
>        * Only one motor is supported. If the gamepad has more than one motor, the higher value of the left and right motor speed applies to both.
>        * On some devices, motor speed control is limited. You can only set the left and right motor speed to either `1.0` (maximum speed) or `0.0` (turned off).
> * Gamepads on iOS, tvOS, and visionOS. Requires Unity 6000.6 or later.
>    * Rumble is supported on gamepads recognized by Apple's GameController framework, including PS4, PS5, Xbox, and Switch Pro controllers.
>    * If the gamepad exposes both motors, you can control each speed individually. Otherwise, the higher value of the left and right motor speed applies to all motors.
>    * The first [`SetMotorSpeeds`](xref:UnityEngine.InputSystem.Haptics.IDualMotorRumble) call allocates rumble resources, which adds latency.

> >    * When all motor speeds are set to `0.0`, an inactivity timer starts. After two minutes, rumble resources are released to preserve controller battery. The next [`SetMotorSpeeds`](xref:UnityEngine.InputSystem.Haptics.IDualMotorRumble) call reallocates them and again incurs allocation latency.

Reviewer: ```suggestion

```
